src/main/java/uk/gov/hmcts/cmc/claimstore/services/CountyCourtJudgmentService.java

```diff
@@ -60,7 +63,10 @@ public Claim save(
 
         eventProducer.createCountyCourtJudgmentEvent(claimWithCCJ, authorisation);
 
-        appInsights.trackEvent(AppInsightsEvent.CCJ_REQUESTED, "referenceNumber", claim.getReferenceNumber());
+        appInsights.trackEvent(CCJ_REQUESTED, "referenceNumber", claim.getReferenceNumber());
+        if (countyCourtJudgment.getCcjType() == CountyCourtJudgmentType.ADMISSIONS) {
+            appInsights.trackEvent(CCJ_REQUESTED_BY_ADMISSION, "referenceNumber", claim.getReferenceNumber());
```

Comment:
The above code will fire two events in case of CCJ by admissions. Do you really want to fire two events here?

Reply:
the other was there already, I can take it out?

Reply:
I think we want to fire one event in both cases. just the event name should be different based on the scenario.